workflow-templates/release-with-moving-tags.yml

```diff
@@ -2,10 +2,15 @@ name: Create release and move major/minor tags
 
 # This workflow creates a release when changes are merged into main
 
+# This is a hack to run on merge of a PR into main
+# as we do not get PR details when triggering on push to main
+# All jobs must be conditional on ${{ github.event.pull_request.merged }}
 on:
-  push:
-    branches:
+  pull_request:
+    branches: 
```

Comment:
is that an intentional extra white space?

Reply:
Nope, removed, thanks